samples/Exporters/Console/TestJaegerExporter.cs

```diff
@@ -50,9 +48,7 @@ internal static object RunWithActivity(string host, int port)
             {
                 sample.Start();
 
-                Console.WriteLine("Traces are being created and exported" +
-                    "to Jaeger in the background. Use Jaeger to view them." +
-                    "Press ENTER to stop.");
+                Console.WriteLine("Sample is running on the background, press ENTER to stop");
```

Comment:
minor: can you use the original message here?

Reply:
i added in another PR :)